pkg/apis/sealed-secrets/v1alpha1/doc.go

```diff
@@ -1,5 +1,5 @@
 // go mod vendor doesn't preserve executable perm bits
-//go:generate bash ../../../../vendor/k8s.io/code-generator/generate-groups.sh all github.com/bitnami-labs/sealed-secrets/pkg/client github.com/bitnami-labs/sealed-secrets/pkg/apis sealed-secrets:v1alpha1 --go-header-file boilerplate.go.txt
+//go:generate bash ../../../../vendor/k8s.io/code-generator/generate-groups.sh deepcopy,defaulter,client,lister github.com/bitnami-labs/sealed-secrets/pkg/client github.com/bitnami-labs/sealed-secrets/pkg/apis sealed-secrets:v1alpha1 --go-header-file boilerplate.go.txt
```

Comment:
I assume this was necessary due to an updated `generate-groups.sh` script? It looks like not using `all` means no `informer` generator, is that correct?

Reply:
yeah, the generated informer is buggy: it generates the file as `package sealed-secret` which then fails to gofmt and aborts.

I created this PR mostly to make use of the CI integration test to check I'm on the right track.

I'll find a way to correctly generate the informer with the new version of the generator before merging this